WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -1,14 +1,16 @@
 import Yosemite
 
 /// Provides data for product form UI, and handles product editing actions.
-final class ProductFormViewModel {
+final class ProductFormViewModel: ProductFormViewModelProtocol {
+    typealias ProductModel = Product
+
     /// Emits product on change, except when the product name is the only change (`productName` is emitted for this case).
     var observableProduct: Observable<Product> {
         productSubject
     }
 
     /// Emits product name on change.
-    var productName: Observable<String> {
+    var productName: Observable<String>? {
```

Comment:
Are we using RX in the project? 😅

Reply:
haha not the whole Rx/Combine, but we do have [`Observable` and `PublishSubject` suppport](https://github.com/woocommerce/woocommerce-ios/tree/058d800ee744f75cd6a0c06bb4f1ec98c4514f1e/WooCommerce/Classes/Tools/Observables) that @shiki added before we can use Combine in iOS 13! 💯 (though after playing with Combine for a little, I find RxSwift much easier to use... like returning an Observable from a Subject with `asObservable()`, creating a derived @Published var, and @ObservedObject is just kind of odd. I think I just haven't got the hang of Combine 🤯 )